styles/README.md

```diff
@@ -20,17 +20,27 @@ CSS is broken down into three major sections:
 * `global.less` primarily includes mix-ins and is unlikely to change much
 * `layout.less` governs page layout details
 * `themes/theme.less` governs colors, fonts, and other things impacted by themes
+  See the [Themes README](themes/README.md) for more information.
 
 ## Changing CSS
-To update CSS, only update the .less files in the styles/ and styles/themes
-directories. The .css files are automatically generated from the .less files.
+To update CSS, only update the .less files in the `styles/` and `styles/themes/`
+directories. The `.css` files are automatically generated from the `.less` files.
 
-To create the .css files, use the Makefile in the SETUP directory:
-```
+To create the `.css` files, use the Makefile in the SETUP directory:
+```bash
 cd SETUP
 make less
 ```
 
-Check in both the .css and .less files. While it isn't ideal to have generated
+Check in both the `.css` and `.less` files. While it isn't ideal to have generated
 code checked into source control, doing so means that consumers of the DP code
 don't also have to install and use less to generate the CSS.
+
+## Style design philosophy and demo
+The `design_philosophy.php` and `style_demo.php` pages discuss some of the
+thinking behind the site's design. You can view these pages rendered at pgdp.net
+at:
+
+* [Design Philosophy](https://www.pgdp.net/c/styles/design_philosophy.php)
```

Comment:
I don't see any recommended changes for the Design Philosophy, but in looking at it, it was written before the current styling was rolled out. Some of the wording for that should probably be tweaked, as well. For example, under Design Goals/Consistency, the last sentence of that para is "This means that many pages will change such that all look alike.". We probably don't need that sentence any more.

Reply:
Good call. I added a commit to update the tenses on that page.